### ISSUE TITLE
[Feature Request] Make mouse cubes not instakill you on consumption

### ISSUE THREAD

**Issue by BomberBro** [d3athrow/vgstation13#29540]
The mice chem has some text about spitting out a mouse when taken, that should be used instead.

**Comment by Eneocho** [comment]
You're eating a cube that expands into a fully sized mice when wet. Mouths and throats are wet with saliva. Mice expands. If you don't want to get cucked by a micecube don't eat the micecube in the first place.

**Comment by BomberBro** [comment]
Then why do monkey cubes not make people's heads explode? If I can spit a mouse out with the mouse chem it's my god given right to do it with cubes.

**Comment by SGuivenne** [comment]
> 
> 
> Then why do monkey cubes not make people's heads explode? If I can spit a mouse out with the mouse chem it's my god given right to do it with cubes.

Monkey Cubes horrifically maim you and generally kill you horribly.

**Comment by BomberBro** [comment]
Yeah but they usually just do that in your stomach. I'm also pretty sure there's a chance to not get damaged at all if you're lucky but that may just be human cubes.
